src/shop/searchFilters/SearchFilters.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Switch } from 'react-native';
+
+import { Button, Typography } from '../../_components';
+import { useSearchFilters } from '../../_context';
+import { TRootStackNavigationProp } from '../../_routing';
+import { theme } from '../../_styles/theme';
+import * as Styled from './style';
+
+type TProps = {
+  navigation: TRootStackNavigationProp<'SearchFilters'>;
+};
+
+export const SearchFilters = ({ navigation }: TProps) => {
+  const { t } = useTranslation();
+  const { getFilter, setFilter, persistChangedFilters, clearChangedFilters } = useSearchFilters();
+
+  navigation.addListener('beforeRemove', () => {
+    clearChangedFilters();
+  });
+
+  function onSubmit() {
+    persistChangedFilters();
+    navigation.pop();
+  }
+
+  return (
+    <Styled.Container contentContainerStyle={{ flex: 1 }}>
```

Comment:
Doet flex: 1 iets? Ik dacht dat (als het voor het stretchen van de content is) dat je flexGrow: 1 moest hebben

Reply:
ja, die stretch de content wel degelijk, maar 'k ga het aanpassen want er valt wel iets te zeggen voor leesbaarheid 👍🏻